core/handler_schema_constants.py

```diff
@@ -0,0 +1,151 @@
+# coding: utf-8
+
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Handlers which require schema validation, but currently they do
+# not have schema. In order to add schema incrementally this list is
+# maintained. Please remove the name of the handlers if they already
+# contains schema.
+
+"""Contains a list of handler class names that do not contain schemas.
+This is a temporary file which will be removed once all of the handlers
+mentioned in the list have a schema. This file resides in this folder as
+it will be used both by  scripts and cores and hence would be easier to
```

Comment:
Please read my previous comment for this line more carefully. In particular, "cores" does not make sense from an English perspective.